packages/composites/medianizer/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,198 @@
+import sinon, { createSandbox } from 'sinon'
+import { assertSuccess, assertError } from '@chainlink/ea-test-helpers'
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Execute, Config } from '@chainlink/types'
+import { util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from '../../src/adapter'
+import { makeConfig } from '../../src/config'
+
+const AdapterStubs: Record<string, any> = {
+  COINGECKO: {
+    jobRunID: '1',
+    data: {
+      result: 1000,
+    },
+    result: 1000,
+    statusCode: 200,
+  },
+  COINPAPRIKA: {
+    jobRunID: '1',
+    data: {
+      result: 2000,
+    },
+    result: 2000,
+    statusCode: 200,
+  },
+  FAILING: false,
+}
+
+const setupEnvironment = (adapters: string[]) => {
+  for (const a of adapters) {
+    process.env[`${a.toUpperCase()}_${util.ENV_ADAPTER_URL}`] = `http://test/${a}`
+  }
+}
+
+describe('medianizer', () => {
+  let execute: Execute
+  let config: Config
+  let sandbox: sinon.SinonSandbox
+  let server: sinon.SinonFakeServer
+
+  beforeEach(() => {
+    execute = makeExecute()
+    config = makeConfig()
+    sandbox = createSandbox()
```

Comment:
Can we use the same testing pattern that's presented in `packages/composites/synth-index/test/integration`? We use nock which lets you record http calls first, which makes it super easy to generate stubs later on

Reply:
Happy to do this in the test improvements of `outlier-detection`, though I don't see it as necessary to get this PR out.